core/transport/src/main/proto/grpc/version.proto

```diff
@@ -20,11 +20,12 @@ enum ServiceType {
   METRICS_MASTER_CLIENT_SERVICE = 9;
   JOB_MASTER_CLIENT_SERVICE = 10;
   JOB_MASTER_WORKER_SERVICE = 11;
-  FILE_SYSTEM_WORKER_WORKER_SERVICE = 12;
+  FILE_SYSTEM_WORKER_WORKER_SERVICE = 12 [deprecated = true];
   JOURNAL_MASTER_CLIENT_SERVICE = 13;
   TABLE_MASTER_CLIENT_SERVICE = 14;
   META_MASTER_BACKUP_MESSAGING_SERVICE = 15;
   RAFT_JOURNAL_SERVICE = 16;
+  BLOCK_WORKER_CLIENT_SERVICE = 17;
 }
```

Comment:
I failed to just rename this enum branch, because [proto-backwards-compat-maven-plugin](https://github.com/salesforce/proto-backwards-compat-maven-plugin) seems to think renaming breaks backward compatibility. This is somewhat surprising, as I thought renaming a field name is not a breaking change in protobuf,

Reply:
Something like 

```
enum EnumAllowingAlias {
    option allow_alias = true;
    UNKNOWN = 0;
    STARTED = 1;
    RUNNING = 1;
  }
```

may be worth a try: https://developers.google.com/protocol-buffers/docs/proto3#enum

Reply:
As it's no longer relevant to the compatibility issue, I just reverted to deprecating the old field. Since there is not much at stake, I'd say we just dicard `FILE_SYSTEM_WORKER_WORKER_SERVICE` rather than keeping it as a valid alias.

Reply:
In this case, you can reserve the value to prevent accidental use in the future: https://developers.google.com/protocol-buffers/docs/proto3#reserved_values

Reply:
That works. I wonder why the plugin is OK with deleting that variant outright🤦‍♂️